src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/UpdateActionPlanActivityDTO.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
```

Comment:
i feel like this would be more at home alongside `ActionPlanActivityDTO` which is actually inside `ActionPlanDTO.kt`. either that or split both out into their own classes?